### ISSUE TITLE
Staging Review finding: Heading hierarchy doesn't match page organization

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128400]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool Team 1
**Product:** Claim Status Tool
**Feature:** Document Submission Status

## Findings details
**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The “What you need to do” section is marked as an H3. The alert that follows also begins with an H3, which makes them siblings in the document structure. However, visually and conceptually, the alert appears to be nested under “What you need to do,” creating a mismatch between the programmatic hierarchy and the perceived hierarchy.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="311" height="327" alt="heading hierarchy" src="https://github.com/user-attachments/assets/a7cad752-45ad-41ae-a600-06d55196604a" />

Correct hierarchy (similar layout scenario):
<img width="302" height="604" alt="correct hierarchy src="https://github.com/user-attachments/assets/bbcad3be-c3c8-4fd0-92f3-93b6936a620e" />

</details>

## Recommended action

Update the alert heading to an H4 to ensure that the document structure and organization is hierarchical and makes sense. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
